src/openrpc/secure_storage.json

```diff
@@ -256,6 +256,291 @@
+        },
+        {
+          "name": "value",
+          "summary": "Value to set",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "options",
+          "summary": "Optional parameters to set",
+          "schema": {
+            "$ref": "#/components/schemas/StorageOptions"
+          },
+          "required": false
+        }
+      ],
+      "result": {
+        "name": "success",
+        "schema": {
+          "const": null
+        }
+      },
+      "examples": [
+        {
+          "name": "Set a refresh token with name authRefreshToken with optional paramter for appId foo",
+          "params": [
+            {
+              "name": "appId",
+              "value": "foo"
+            },
+            {
+              "name": "scope",
+              "value": "device"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            },
+            {
+              "name": "value",
+              "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
+            },
+            {
+              "name": "options",
+              "value": { 
+                "ttl": 600
+              }
+            }
+          ],
+          "result": {
+            "name": "defaultResult",
+            "value": null
+          }
+        },
+        {
+          "name": "Set a refresh token with name authRefreshToken without optional parameter for appId foo",
+          "params": [
+            {
+              "name": "appId",
+              "value": "foo"
+            },
+            {
+              "name": "scope",
+              "value": "account"
+            },
+            {
+              "name": "key",
+              "value": "authRefreshToken"
+            },
+            {
+              "name": "value",
+              "value": "VGhpcyBub3QgYSByZWFsIHRva2VuLgo="
+            }
+          ],
+          "result": {
+            "name": "defaultResult",
+            "value": null
+          }
+        }
+      ]
+    },
+    {
+      "name": "removeForApp",
+      "summary": "Removes single data value for a specific app.",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-manages": [
+            "xrn:firebolt:capability:storage:secure"
+          ]
+        }
+      ],
+      "params": [
+        {
+          "name": "appId",
+          "summary": "appId for which values are removed",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        },
+        {
+          "name": "scope",
+          "summary": "The scope of the key/value",
+          "schema": {
+            "$ref": "#/components/schemas/StorageScope"
+          },
+          "required": true
+        },
+        {
+          "name": "key",
+          "summary": "Key to remove",
+          "schema": {
+            "type": "string"
+          }
+        }
```

Comment:
key should be required, right? we don't want people using this api to clear all of the keys in app/scope, that's what clear is for.